test/commands/use.test.ts

```diff
@@ -2,15 +2,18 @@ import UseCommand from '../../src/commands/use'
 import * as fs from 'fs-extra'
```

Comment:
In this file, can we assert that no network requests were made? I think we can do this by just asserting that `downloadAndExtract` was not called